frontend/app/views/event/list.scala.html

```diff
@@ -16,6 +15,7 @@
                 <h3 class="listing__title h-intro">Featured</h3>
             </div>
             <div class="listing__content">
+                <ul>
```

Comment:
You can remove both `ul` items, this doesn't need a list around it for the hero item (as it's just one full width block)